### ISSUE TITLE
Reboot when starting certain apps

### ISSUE THREAD

**Issue by plata** [AICP/device_sony_castor#1]
When I start certain apps (e.g. Firefox Preview, GitHub), the device reboots.

Version: aicp_castor_p-14.0-WEEKLY-20200403 (latest official)

**Comment by plata** [comment]
Might be https://github.com/mozilla-mobile/fenix/issues/5663 (I do not get a boot loop though).

**Comment by SpiritCroc** [comment]
I can not reproduce this issue on AICP 15.0. AICP 14.0 is no longer supported.

**Comment by plata** [comment]
@SpiritCroc if 14.0 is not supported anymore, why is it the version on https://dwnld.aicp-rom.com/? Where should I get 15.0? According to [XDA](https://forum.xda-developers.com/xperia-z2-tablet/orig-development/rom-aicp-15-0-q10-0-castor-windy-sony-t4047535), the 15.0 builds are not official and will not receive any OTA updates.

**Comment by SpiritCroc** [comment]
Since there are some minor issues remaining with the AICP 15.0 test builds (to be found at the xda thread you linked) and I don't have much time to take fix them unfortunately, there have been no official 15.0 builds yet, and thus the latest official build is still 14.0.

**Comment by plata** [comment]
The issues for the 15.0 builds seem to be very minor (and less severe than this one for 14.0). Wouldn't it make sense to have official builds for 15.0 with it's current state instead? Right now it even seems that you cannot download any version from the official website (even though the device is still listed).

**Comment by SpiritCroc** [comment]
Maybe, I'll think about it

**Comment by SpiritCroc** [comment]
Have you tried whether AICP 15.0 actually fixed your issue?

**Comment by plata** [comment]
Yes, Firefox Preview works on 15.0.

The only bug I've noticed so far is that the orientation always switches to landscape mode in the app overview (like described on XDA). I noticed that it first uses the correct orientation shortly and then switches to the default (like autorotate is off). Would be nice if it could get fixed but this is a minor issue really.

**Comment by Nolimit-SS** [comment]
I know 14 isn't supported anymore but 15 doesn't work well for my device and firefox, regular one from playstore, causes device restarts like mentioned here. I read the other thread about kernel panic and I'd love if someone could help me incorporate the fix into my aicp14 because I use firefox on several devices and it's important to me to have it working. 
I downgraded to some older version I found but that will lead to other problems down the road.
